Hw1/Task1/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿public class Program {
+    public static void Main(string[] args) {
+        Console.Write("Put your first number: ");
+        int first = int.Parse(Console.ReadLine() ?? "0");
+        Console.Write("put your second number: ");
+        int second = int.Parse(Console.ReadLine() ?? "0");
+
+        for(int i=first; i<=second; i++) {
+            int processed = i;
```

Comment:
processed can be negative (int), please fix this case

Reply:
I do not see a problem with processed being negative. But i changed code so that it takes under consideration also negative values that have two A's in it.